src/transformers/models/jukebox/configuration_jukebox.py

```diff
@@ -0,0 +1,667 @@
+        mask (`bool`, *optional*, defaults to False):
+            Whether or not to mask the previous positions in the attention.
+        max_duration (`int`, *optional*, defaults to 600):
+            _description_
+        max_nb_genres (`int`, *optional*, defaults to 1):
+            _description_
+        merged_decoder (`bool`, *optional*, defaults to True):
+            Whether or not the decoder and the encoder inputs are merged. This is used for the seperated
+            encoder-decoder architecture
+        metadata_conditioning (`bool`, *optional*, defaults to True):
+            _description_
+        metadata_dims (`tuple(int)`, *optional*, defaults to (604, 7898)):
+            Number of genres and the number of artists that were used to train the embedding layers of the prior
+            models.
+        min_duration (`int`, *optional*, defaults to 0):
+            _description_
+        mlp_multiplier (`float`, *optional*, defaults to 1.0):
+            Multiplier coefficient used to define the hidden dimension of the MLP layers. 0.25 means that 0.25*width of
+            the model will be used.
+        n_ctx (`int`, *optional*, defaults to 6144):
+            Number of context tokens for each prior. The context tokens are the music tokens that are attended to when
+            generating music tokens.
+        n_heads (`int`, *optional*, defaults to 2):
+                Number of attention heads.
+        nb_relevant_lyric_tokens (`int`, *optional*, defaults to 384):
+            Number of lyric tokens that are used when sampling a single window of length `prior_n_ctx`
+        res_conv_depth (`int`, *optional*, defaults to 3):
+            Depth of the `JukeboxDecoderConvBock` used to upsample the previously sampled audio in the
+            `JukeboxMusicTokenConditioner`.
+        res_conv_width (`int`, *optional*, defaults to 128):
+            Width of the `JukeboxDecoderConvBock` used to upsample the previously sampled audio in the
+            `JukeboxMusicTokenConditioner`.
+        res_convolution_multiplier (`int`, *optional*, defaults to 1):
+            Multiplier used to scale the `hidden_dim` of the `JukeboxResConv1DBlock`.
+        res_dilation_cycle (`int`, *optional*, defaults to None):
+            Dilation cycle used to define the `JukeboxMusicTokenConditioner`. Usually similar to the ones used in the
+            corresponding level of the VQVAE. The first prior does not use it as it is not conditioned on upper level
+            tokens.
+        res_dilation_growth_rate (`int`, *optional*, defaults to 1):
+            Dilation grow rate used between each convolutionnal block of the `JukeboxMusicTokenConditioner`
+        res_downs_t (`tuple(int)`, *optional*, defaults to (3, 2, 2)):
+            Downsampling rates used in the audio conditioning network
+        res_strides_t (`tuple(int)`, *optional*, defaults to (2, 2, 2)):
```

Comment:
```suggestion
        res_downs_t (`tuple(int)`, *optional*, defaults to `(3, 2, 2)`):
            Downsampling rates used in the audio conditioning network
        res_strides_t (`tuple(int)`, *optional*, defaults to `(2, 2, 2)`):
```
Same comment on switch to list.